docs/tutorials/FiveMinuteIntro.md

```diff
@@ -4,9 +4,11 @@
 
 * A Unix-based operating system (yes, that includes Mac!)
 * A Java 11 runtime environment 
-	* You can see what you have by running `$ java -version` on a terminal. 
-	* If not, you can download Java [here](https://adoptopenjdk.net/).
-	* You might need to update the `export JAVA_HOME` in your bash profile to point to your JAVA install location.
+    * You can see what you have by running `$ java -version` on a terminal. You're looking for a version that's at least `1.11` or higher.
```

Comment:
Don't think this is true, newer major releases of Java are not well-supported yet. Also, `1.11` should just be `11` as of... [2004 it looks like](https://en.wikipedia.org/wiki/Java_version_history#Release_table).